authorization/auth_cloudSolutionProvider.md

```diff
@@ -0,0 +1,62 @@
+    ```
+
+## Token acquisition flows
+Partner managed apps are identical to regular multi-tenant apps in virtually all respects. As such, the token acquisition flows <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-code" target="_newtab">authorization code grant flow</a> or the <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-service-to-service" target="_newtab">service to service client credentials flow</a> are exactly the same.
+
+Apart from pre-consented access to all your customer tenants, there is one other capability afforded to partner managed apps. It allows for your agents to use your app to access your customers' tenant data (delegated admin privileges). Conceptually your agent signs in to your app with their user credentials from your partner tenant, but your app then acquires an access token to one of your partner managed customers' tenants, and uses that to call Microsoft Graph. This is a slight variation on the <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-code" target="_newtab">authorization code grant flow</a>. 
+
+To see how this would look, imagine your partner tenant is *partner.com* (which is the home tenant for your agents) and one of your customers is *customer.com*:
+
+1. The first step to <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-code#request-an-authorization-code" target="_newtab">acquire an authorization code</a> is exactly the same, where your app makes a request to the ```/authorize``` endpoint, specifying ```common``` or ```partner.com``` for the tenant.
+
+   ```code
+   GET https://login.microsoftonline.com/common/oauth2/authorize
+   ```
+
+2. In the second step, where your app <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-code#use-the-authorization-code-to-request-an-access-token" target="_newtab">aquires an access token using the authorization code</a>, you must use a **customer tenant** as the target tenant when making the request to the ```token``` endpoint.
+
+   ```code
+   POST https://login.microsoftonline.com/customer.com/oauth2/token
+   ```
+
+## Other considerations
+Partner managed apps are currently limited to a single region.  It is expected that you should have a separate tenant for each region you sell in.  For example, if your partner managed app is registered in a tenant in the US but your customer is in the EU – the partner managed app will not work.  Each of your regional partner tenants must maintain their own set of partner managed apps to manage customers within the same region. This may require additional logic in your app (prior to sign-in) to get your customers' sign-in username to decide which region-specific partner managed app identity to use, to serve the user.
```

Comment:
For more concise writing, remove "It is expected". Change "may" to "might" and lowercase Logic.